_targets/build.project.armv7a7-imx6ull

```diff
@@ -112,38 +112,40 @@ b_syspage_gen() {
 b_image_target() {
 	b_log "Creating image from $PREFIX_ROOTFS"
 
-	b_mkscript_user  "${UPLOAD_SCRIPT[@]}"
+	b_mkscript_user "${UPLOAD_SCRIPT[@]}"
 	b_mkscript_preinit
 
 	# Build production image
 	b_log "RAM Image"
-	b_kernel_image "$PREFIX_BOOT/phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.img"
-	b_syspage_gen "$PREFIX_BOOT/phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.img" "$PREFIX_BUILD/plo/user.plo"
+	b_kernel_image "$PREFIX_BOOT/phoenix.img"
```

Comment:
We have viariable with phoenix.img path

Reply:
`phoenix.img` is created by `b_prod_image` which isn't called for `imx6ull` due to lack of `plo` support.